kettle/make_json.py

```diff
@@ -146,6 +147,8 @@ def row_for_build(path, started, finished, results):
         build['started'] = int(started['timestamp'])
         if 'node' in started:
             build['executor'] = started['node']
+        build['repo_commit'] = started.get('repo-commit', started.get('repo-version'))
+        build['repos'] = started.get('repos')
```

Comment:
I recognize there is a dearth of comments here already, so you're just following the pattern.  But I think it might be useful for future maintainers to comment where these are expected to come from, eg: a link back to the testgrid metadata struct or something

Reply:
+1 I will try to make some back reference.